api/management/commands/ingest_icrc.py

```diff
@@ -36,32 +36,36 @@ def handle(self, *args, **kwargs):
         response.raise_for_status()
         soup = BeautifulSoup(response.content, "html.parser")
 
-        # Get the countries information from the "Where we work" page
-        regions_list = soup.find("div", {"id": "blockRegionalList"}).find_all("ul", {"class": "list"})
+        # Get countries information from "Where we work" page
+        countries = soup.find("div", {"class": "js-select-country-list"}).find("ul").find_all("ul")
+
         country_list = []
-        for region in regions_list:
-            for country in region.find_all("li", {"class": "item"}):
-                # Get key information
+        for region in countries:
```

Comment:
can we write this to `for region in regions_and_countries` or `for region in regions`  ? or give some meaningful name.